rollup/plugins/npmPlugins.js

```diff
@@ -111,15 +111,11 @@ export function makeDebugBuildStatementReplacePlugin() {
  * b) can easily be modified by our users' bundlers to evaluate to false, facilitating the treeshaking of logger code.
  *
  * When `undefined` is passed,
- * end users can define e.g. `__SENTRY_EXCLUDE_CANVAS__` in their bundler to shake out canvas specific rrweb code.
+ * end users can define e.g. `__RRWEB_EXCLUDE_SHADOW_DOM__` in their bundler to shake out shadow dom specific rrweb code.
  */
-export function makeRrwebBuildPlugin({ excludeCanvas, excludeShadowDom, excludeIframe } = {}) {
+export function makeRrwebBuildPlugin({ excludeShadowDom, excludeIframe } = {}) {
   const values = {};
 
-  if (typeof excludeCanvas === 'boolean') {
-    values['__RRWEB_EXCLUDE_CANVAS__'] = excludeCanvas;
-  }
-
```

Comment:
Don't we have to keep this and set it to false?

Reply:
No. in 2.2.0 this is excluded by default. Instead, if you want this you have to do:

```js
import { record, getCanvasManager } from '@sentry-internal/rrweb';

record({
  getCanvasManager
});
```

So it is opt-in and thus we can properly tree shake this. Eventually we'll expose this as some API to replay (*when we actually support _replaying_ canvas, which we don't do yet)

Reply:
ah gotcha. Thanks for explaining!